plugins/inputs/sqlserver/README.md

```diff
@@ -46,6 +46,11 @@ GO
   #  "Server=192.168.1.10;Port=1433;User Id=<user>;Password=<pw>;app name=telegraf;log=1;",
   # ]
 
+  ## This enables a specific set of queries depending on the database type. If specified, it replaces azuredb = true/false and query_version = 2
```

Comment:
I would point out that "query_version" is now outdated/deprecated and "database_type" should be used instead.
I'll also edit its description in order to match version 2 current state.

```md
  ## The "query_version" parameter is deprecated, use the new "database_type" parameter.
  ## maintained for retro-compatibility reasons,version 1 is no longer mantained, therefore it's suggested to use version 2
  ## version 2 supports SQL Server from SQL 2008 SP3 and SQL 2008 R2, and the cloud version of SQL Server AzureSQLDB and Managed Instance
  query_version = 2
```

Reply:
Done, also added deprecation notice in log file.